src/main/java/io/github/jhipster/online/service/JdlMetadataService.java

```diff
@@ -148,7 +148,7 @@ public void updateJdlContent(JdlMetadata jdlMetadata, String content) throws Fil
         } else if (jdlMetadata.isPresent() && jdlMetadata.get().getUser().equals(this.getUser())) {
             return jdlMetadata;
         } else {
-            throw new AccessDeniedException("Current user does not have access to this JDL file");
```

Comment:
I think this change should be reverted. The alignment seems removed here. 🤔

Reply:
we should add prettier-java to solve this kind of formatting

Reply:
@pascalgrimaud : Good idea; I've added it at https://github.com/jhipster/jhipster-online/pull/249.